tensorflow/contrib/opt/python/training/external_optimizer.py

```diff
@@ -397,10 +397,6 @@ def loss_grad_func_wrapper(x):
             'automatically and cannot be injected manually'.format(kwarg))
 
     minimize_kwargs.update(optimizer_kwargs)
-    if method == 'SLSQP':
```

Comment:
Please add a unit test that uses SLSQP.

Reply:
Can you point me to some documentation or example on how to add unit test in tensorflow?

Reply:
The easiest is probably to clone one of the existing tests for this module here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/opt/python/training/external_optimizer_test.py

Reply:
Thanks. I took a look at the file you provided. Actually there is already a unit test that uses SLSQP.

Reply:
I mean, can you add a test that uses SLSQP with a step callback. Every change should have a unit test to make sure that the intended behavior is working.